recipes/linkage2allegro/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: linkage2allegro
+  version: "2017.1"
+
+source:
+  url: https://github.com/BioTools-Tek/linkage-converter/archive/d8eefb76ad2cf3ab5fb6b190e968f6f933cc2886.tar.gz
```

Comment:
please include: `fn: linkage2allegro-2017.1.tar.gz`

Reply:
@druvus: Out of curiosity: Why is the explicit filename preferred? For debugging and/or archival purposes from, e.g., Galaxy's `cargo-port`, or something else?